### PR TITLE
Fix typos and broken links in docs

### PR DESCRIPTION
**Pull request by yujuhong**

### CODE REVIEW

**Comment by Random-Liu** [comment]
@yujuhong Please sign your commit `git commit --amend -s`. :)
And thanks for fixing this! :)

File: docs/crictl.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ or `crictl` please make sure the issue has not already been submitted.
 If you have not already installed crictl please install the version compatible
 with the `cri-containerd` you are using. If you are a user, your deployment
 should have installed crictl for you. If not, get it from your release tarball.
-If you are a developer the current version of crictl is specified [here](./hack/version).
+If you are a developer the current version of crictl is specified [here](./hack/versions).
```
Comment (Random-Liu) [inline docs/crictl.md]: It should be `../hack/versions`.

Comment (Random-Liu) [reply docs/crictl.md]: @yujuhong Can you also address this?

Comment (yujuhong) [reply docs/crictl.md]: I thought I did, but forgot to amend. Pushed again

**Comment by Random-Liu** [comment]
@yujuhong Thanks! :D 
/lgmt

**Pull request merged**